ts/src/pro/bitvavo.ts

```diff
@@ -506,6 +531,512 @@ export default class bitvavo extends bitvavoRest {
+        const request = this.cancelOrderRequest (id, symbol, params);
+        return await this.watchRequest ('privateCancelOrder', request);
+    }
+
+    async cancelAllOrdersWs (symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitvavo#cancelAllOrdersWs
+         * @see https://docs.bitvavo.com/#tag/Orders/paths/~1orders/delete
+         * @description cancel all open orders
+         * @param {string} symbol unified market symbol, only orders in the market of this symbol are cancelled when symbol is not undefined
+         * @param {object} [params] extra parameters specific to the bitvavo api endpoint
+         * @returns {object[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        await this.loadMarkets ();
+        await this.authenticate ();
+        const request = {};
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['market'] = market['id'];
+        }
+        return await this.watchRequest ('privateCancelOrders', this.extend (request, params));
+    }
+
+    handleMultipleOrders (client: Client, message) {
+        //
+        //    {
+        //        action: 'privateCancelOrders',
+        //        response: [{
+        //            orderId: 'd71df826-1130-478a-8741-d219128675b0'
+        //        }]
+        //    }
+        //
+        const action = this.safeString (message, 'action');
+        const response = this.safeValue (message, 'response');
+        const firstRawOrder = this.safeValue (response, 0, {});
+        const marketId = this.safeString (firstRawOrder, 'market');
+        const orders = this.parseOrders (response);
+        let messageHash = this.buildMessageHash (action, { 'market': marketId });
+        client.resolve (orders, messageHash);
+        messageHash = this.buildMessageHash (action, message);
+        client.resolve (orders, messageHash);
+    }
+
+    async fetchOrderWs (id: string, symbol: string = undefined, params = {}) {
+        /**
+         * @method
+         * @name bitvavo#fetchOrderWs
+         * @description fetches information on an order made by the user
+         * @param {string} symbol unified symbol of the market the order was made in
+         * @param {object} [params] extra parameters specific to the bitvavo api endpoint
+         * @returns {object} An [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrder() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        await this.authenticate ();
+        const market = this.market (symbol);
+        const request = {
+            'orderId': id,
+            'market': market['id'],
+        };
+        return await this.watchRequest ('privateGetOrder', this.extend (request, params));
```

Comment:
@pcriadoperez shouldn't we make the `id` /`symbol` part of the `messageHash` ?  what if the user does
```Javascript
Promise.all(ex.fetchOrderWs('id1'), ex.fetchOrderWs('id2'))
```